.travis.yml

```diff
@@ -17,6 +17,9 @@ env:
 install:
   - git clone --depth 1 https://github.com/mfem/mfem.git
   - make -C mfem -j2 serial MFEM_CXXFLAGS=-O
+  - git clone --depth 1 https://github.com/libocca/occa.git
+  - make -C occa -j2
+  - export OCCA_DIR=$PWD/occa LD_LIBRARY_DIR+=":${PWD}/lib"
```

Comment:
What is `LD_LIBRARY_DIR`?

Reply:
Also, that path doesn't look like it is anything. I expect you can delete that part.

Reply:
Its in the OCCA build instructions. I sometimes have trouble locally when I skip it.

Reply:
It's where `libocca.so` lives

Reply:
`LD_LIBRARY_PATH` is in the build instructions. (`LD_LIBRARY_DIR` isn't a thing as far as I know.) LibCEED uses RPATH to link so the environment variable isn't needed. The library is in `$OCCA_DIR/lib`, not `$PWD/lib`.

Reply:
Should be all good now. Build times are much more reasonable today